src/app.ts

```diff
@@ -32,7 +40,48 @@ const validateMfaExpiry = (mfaExpiry: number): boolean | string => {
   }
 };
 
-const switchProfile = async (name?: string, forceMFA?: boolean): Promise<void> => {
+const switchToAssumeRoleProfile = async (
```

Comment:
I'd move this function below `switchProfile`. Also maybe the name should be `switchAssumeRoleProfile` for consistency?

Reply:
`switchProfile` actually depends on `switchToAssumeRoleProfile`, so the linter would throw a fit. I could change the name though.